product_variant_sale_price/models/product_product.py

```diff
@@ -34,20 +34,18 @@ def _compute_lst_price(self):
         uom_model = self.env["uom.uom"]
         for product in self:
             price = product.fix_price or product.list_price
-            if "uom" in self.env.context:
-                price = product.uom_id._compute_price(
-                    price, uom_model.browse(self.env.context["uom"])
-                )
+            if self.env.context.get("uom"):
+                context_uom = uom_model.browse(self.env.context.get("uom"))
```

Comment:
**[suggestion]** Since you already checked that "uom" key is in the context and has a value, you can access direct the dictionary like self.env.context["uom"] instead of using get, since it is faster.